runtime/compiler/aarch64/codegen/ARM64PrivateLinkage.cpp

```diff
@@ -1547,12 +1547,13 @@ static bool getProfiledCallSiteInfo(TR::CodeGenerator *cg, TR::Node *callNode, u
  * @param[in] profiledMethod: method suggested by interpreter profiler
  * @param[in] vftReg:         register containing VFT
  * @param[in] tempReg:        temporary register
+ * @param[in] temp2Reg:        temporary register
```

Comment:
- Please adjust the indentation of `temporary register`.
- `slotCount` is missing as a param in this comment.

Reply:
Adjusted the indentation. Added `slotCount` to the comment.